### PR TITLE
Allow `ApplicationInstallation` config field to be blank

### PR DESCRIPTION
**Pull request by delenamalan**
When it's not blank, when we run `full_clean` on an `ApplicationInstallation` which doesn't have any config fields (i.e. no config form), we get an error:

```
ValidationError: {'config': ['This field cannot be blank.']}
```

### CODE REVIEW

**Review by hasier — COMMENTED** [review]
I believe there is no need to update the field, currently we already have integrations installed with [an empty config](https://one.staging1.yoyowallet.com/admin/login/applicationinstallation/?p=16) :thinking:

**Comment by delenamalan** [comment]
> I believe there is no need to update the field, currently we already have integrations installed with an empty config

I'm not sure why Django admin lets the models be saved, but running `full_clean()` fails on them.

File: drf_integrations/models.py [hunk 1]
```diff
@@ -270,7 +270,7 @@ class Meta:
             on_delete=models.PROTECT,
             related_name="installations",
         )
-        config = fields.JSONField(null=True)
+        config = fields.JSONField(null=True, blank=True)
```
Comment (gryevns) [inline drf_integrations/models.py:273]: I think Django will create a migration for this which is missing from the PR.

Comment (delenamalan) [reply drf_integrations/models.py:273]: Thank you for the reminder Gary!

**Review by gryevns — APPROVED** [review]

**Pull request merged**